.github/workflows/build.yml

```diff
@@ -3,6 +3,10 @@ name: Build
 on:
   pull_request:
     branches: [ master ]
+    paths:
+      - "build-farm/**"
+      - "sbin/**"
+      - "**.sh"
```

Comment:
You should also include this specific workflow file (so we can test changes made to it)

Should just be able to add .github/workflows/build.yml as a path.